hack/generate-operator-bundle.py

```diff
@@ -117,7 +117,9 @@ def generate_csv_base(version, prev_version, hive_image):
     csv['spec']['replaces'] = "hive-operator.v%s" % prev_version
 
     # Update the deployment to use the defined image:
-    image_ref = "%s:v%s" % (hive_image, version)
+    image_ref = hive_image
+    if not ":" in image_ref:
```

Comment:
i'm not sure the best way to check a ref for a tag

Reply:
`git describe --exact-match --tags <ref>` perhaps?

Reply:
Thanks! -- not git tag (English needs more words)

hive_image is a user-specified image ref e.g. `quay.io/gshereme/hive` and I need to know if there's a tag on the end e.g. `quay.io/gshereme/hive:sometag`. If there's not, use the version as the tag.

Reply:
Since its a string these seems reasonable to me.